client/platform/desktop/backend/platforms/windows.ts

```diff
@@ -1,3 +1,224 @@
 /**
  * VIAME process manager for windows platform
  */
+import { app } from 'electron';
+import npath from 'path';
+import { spawn } from 'child_process';
+import fs from 'fs-extra';
+import { xml2json } from 'xml-js';
```

Comment:
```suggestion
import npath from 'path';
import { spawn } from 'child_process';
import { app } from 'electron';
import fs from 'fs-extra';
import { xml2json } from 'xml-js';
```

Reorder imports.

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md